docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -37,6 +37,24 @@ using an import, a task can notify any of the named tasks within the imported fi
 
 To achieve the results of notifying a single name but running mulitple handlers, utilize ``include_tasks``, or ``listen`` :ref:`handlers`.
 
+Jinja Undefined values
+----------------------
+
+Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an exception. This means that you no longer need to jump
```

Comment:
```suggestion
Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an exception. This means you can
```